Apps/APN/ios/Podfile

```diff
@@ -10,7 +10,7 @@ target 'SampleApp' do
   # Flags change depending on the env values.
   flags = get_default_flags()
 
-  pod 'CustomerIO/MessagingPushAPN', '~> 2'
+  pod 'customerio-reactnative/apn', :path => '../node_modules/customerio-reactnative'
```

Comment:
Customers will replace `pod 'CustomerIO/MessagingPushAPN', '~> 2'` in their `Podfile` with an entry like this to install the native iOS SDK.